data/remnant/remnant 1 introduction.txt

```diff
@@ -3655,6 +3655,7 @@ ship "_Ion Timer Ship"
 	# The _Ion Storm Timer: Generator outfit counteracts this until there is an ion storm
 	# sprite "ship/shuttle" # uncomment this for debugging
 	attributes
+		category "Game Mechanics"
```

Comment:
```suggestion
		category "Unclassified"
```

Reply:
This is explicitly used for the mechanic of the ion storm to operate. I wouldn't say this is "unclassified", it is quite literally used for the function. If we are using something that is to operate a mechanic, it is a game mechanic in classification, despite it being an "unclassified" natural body.